winml/test/concurrency/ConcurrencyTests.cpp

```diff
@@ -0,0 +1,283 @@
+#include "testPch.h"
+
+#include "concurrencytests.h"
+#include "model.h"
+#include "SqueezeNetValidator.h"
+#include "threadPool.h"
+
+#include <chrono>
+#include <fstream>
+#include <ctime>
+#include <cstdlib>
+
+using namespace winrt::Windows::AI::MachineLearning;
+using namespace winrt;
+
+namespace
+{
+void LoadBindEvalSqueezenetRealDataWithValidationConcurrently()
+{
+    // Bug 21617097: WinML concurrency test LoadBindEvalSqueezenetRealDataWithValidationConcurrently is disabled due to #21617041
```

Comment:
I don't think we want links to internal bugs in public repo. The internal tracking bug is enough, we can remove the comment here.